src/mono/mono/metadata/class-internals.h

```diff
@@ -529,7 +529,7 @@ mono_generic_param_owner (MonoGenericParam *p)
 	return p->owner;
 }
 
-static inline int
+static inline guint16
```

Comment:
Check callsites, called and used in formatting masks with %d, format mask should either be changed to comply with shorter type or cast it to int at callsite.